pinot-query-planner/src/test/resources/queries/OrderBy.json

```diff
@@ -0,0 +1,58 @@
+{
+  "order_by_planning_tests": {
```

Comment:
If not too much work, we may want to augment these tests to cover alias (AS) as well. Not sure if that is covered elsewhere

Reply:
I was trying to see how alias works with WindowFunctions and looks like it doesn't work! The plan looks no different than if the alias is not present. Will need to look into this. I will add a few alias tests for the other types where it does work.

Reply:
I have added alias tests for all the plan files. Some of the `WindowFunctionPlans` are behaving weird in terms of the alias and others seem to work fine. I've added TODOs to look into why. Let me know if you think this needs further digging before we decide to merge.